docs/concepts/automations.md

```diff
@@ -132,6 +90,205 @@ Prefect tries to infer the relevant event whenever possible, but sometimes one d
 
 Specify a name and, optionally, a description for the automation.
 
+## Custom triggers
+
+Custom triggers allow advanced configuration of the conditions on which a trigger executes its actions. Several custom trigger fields accept values that end with trailing wildcards, like `"prefect.flow-run.*"`.
+
+![Viewing a custom trigger for automations for a workspace in Prefect Cloud.](/img/ui/automations-custom.png)
+
+The schema that defines a trigger is as follows:
+
+| Name               | Type               | Supports trailing wildcards | Description |
+| ------------------ | ------------------ | --------------------------- | ----------- |
+| **match**          | object             | :material-check:            | Labels for resources which this Automation will match. |
```

Comment:
nit: can all these `Name` fields be formatted as inline code? e.g. `match`?

Reply:
I had them that way, but mkdocs will wrap them onto multiple lines if they're code. I couldn't figure out how to fix that